providers/gcp/resources/gcp.lr

```diff
@@ -102,7 +102,7 @@ gcp.project @defaults("name") {
   services() []gcp.service
   // List of recommendations
   recommendations() []gcp.recommendation
-  // GCP GKE resources
+  // Google GKE resources
```

Comment:
I'm just curious why no "cloud" here.

Reply:
```suggestion
  //  Google Kubernetes Engine (GKE)  resources
```

Reply:
Let me revert that one. I wasn't tackling the property descriptions